app/sim_test.go

```diff
@@ -273,12 +274,24 @@ func testAndRunTxs(app *App, config simulation.Config) []simulation.WeightedOper
 				var v int
 				ap.GetOrGenerate(app.cdc, OpWeightMsgPlaceBid, &v, nil,
 					func(_ *rand.Rand) {
-						v = 10000 // TODO
+						v = 100 // TODO
 					})
 				return v
 			}(nil),
 			auctionsimops.SimulateMsgPlaceBid(app.accountKeeper, app.auctionKeeper),
 		},
+		// Pricefeed
+		{
+			func(_ *rand.Rand) int {
+				var v int
+				ap.GetOrGenerate(app.cdc, OpWeightMsgPlaceBid, &v, nil,
+					func(_ *rand.Rand) {
+						v = 10000 // TODO
+					})
+				return v
+			}(nil),
+			pricefeedsimops.SimulateMsgUpdatePrices(app.accountKeeper, app.pricefeedKeeper),
```

Comment:
Can drop `app.accountKeeper` as it isn't being used in `SimulateMsgUpdatePrices()`. I don't think pricefeed sims will need access to account balances since `MsgPostPrice` doesn't trigger any value transfers.

Reply:
🆗